app/lib/main.dart

```diff
@@ -89,31 +89,28 @@ class EffektioHome extends StatefulWidget {
 
 class _EffektioHomeState extends State<EffektioHome>
     with SingleTickerProviderStateMixin {
-  late Future<Client> _client;
+  late Future<Client> client;
   int tabIndex = 0;
-  late TabController _tabController;
-  CrossSigning? _crossSigning;
-  bool isLoading = false;
+  late TabController tabController;
+  CrossSigning? crossSigning;
 
   @override
   void initState() {
     super.initState();
-    _client = makeClient();
-    _tabController = TabController(length: 5, vsync: this);
-    _tabController.addListener(() {
-      setState(() {
-        tabIndex = _tabController.index;
-      });
+    client = makeClient();
+    tabController = TabController(length: 5, vsync: this);
+    tabController.addListener(() {
+      setState(() => tabIndex = tabController.index);
     });
   }
 
   @override
   void dispose() {
-    super.dispose();
-    _crossSigning?.dispose();
+    crossSigning?.dispose();
     Get.delete<ChatListController>();
     Get.delete<ChatRoomController>();
     Get.delete<ReceiptController>();
+    super.dispose();
```

Comment:
It is indeed correct way to order. So it can dispose all resources before calling widget lifecycle end state.

Reply:
Yeah, I met debugging error there so I fixed it.